### ISSUE TITLE
Installation crashing

### ISSUE THREAD

**Issue by philippkayser** [polonel/trudesk#455]
**Is this a BUG REPORT or FEATURE REQUEST?**:
- [X] BUG 
- [ ] FEATURE

**What happened**:
Installing with latest version 1.1.10 will crash while creating admin

`{"success":false,"error":{"errors":{"value":{"name":"ValidatorError","message":"Path `value` is required.","properties":{"message":"Path `value` is required.","type":"required","path":"value"},"kind":"required","path":"value"}},"_message":"settings validation failed","name":"ValidationError","message":"settings validation failed: value: Path `value` is required."}}`

**What did you expect to happen**:
finishing installion

**How to reproduce it (as minimally and precisely as possible)**:
run it via docker
```
docker run --name mongodb \
    -v /tmp/trudesk/data/db:/data/db \
    -v /tmp/trudesk/data/configdb:/data/configdb \
    -d mongo:3.6
```

```
 docker run --name trudesk --link mongodb:mongodb \
    -v /tmp/trudesk/data/trudesk/uploads:/usr/src/trudesk/public/uploads \
    -v /tmp/trudesk/data/trudesk/plugins:/usr/src/trudesk/plugins \
    -v /tmp/trudesk/data/trudesk/backups:/usr/src/trudesk/backups \
    -e NODE_ENV=production \
    -e TRUDESK_DOCKER=true \
    -e TD_MONGODB_SERVER=mongodb -e TD_MONGODB_DATABASE=trudesk \
    -p 50446:8118 \
    -P -d polonel/trudesk:1.1.10
```

**Anything else we need to know?**:

**Environment**:
- Trudesk Version: 1.1.10
- OS (e.g. from /etc/os-release):
- Ubuntu 18.04
- Node.JS Version:
- MongoDB Version: 3.6
- Is this hosted on cloud.trudesk.io: no

**Comment by polonel** [comment]
I'm looking into the issue. This seems to only affect the docker version. I have been successful with the Ubuntu install script.

**Comment by w0vka** [comment]
i have the same error message with the latest image

**Comment by dessy4oko** [comment]
If it`s still problem for you I described a solution here https://github.com/polonel/trudesk/issues/453#issue-1046845925

**Comment by polonel** [comment]
This was fixed in [v1.1.11](https://github.com/polonel/trudesk/releases/tag/v1.1.11)
